src/style.css

```diff
@@ -0,0 +1,88 @@
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Avenir", Arial, sans-serif;
+}
+
+.form {
+  position: relative;
+  margin: 20px 0;
+}
+
+.input {
+  padding: 0;
+  outline: none;
+  width: 100%;
+  font-family: inherit;
+  font-weight: 300;
+  color: #3d4e61;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.input::placeholder {
+  font-family: inherit;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.input:hover {
+  box-shadow: 1px 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  font-weight: 900;
+}
+
+.icon-big {
+  display: block;
+  position: absolute;
+  left: 26px;
+  top: 25px;
+  width: 19px;
+  height: 19px;
+}
+
+.icon-small {
+  display: block;
+  position: absolute;
```

Comment:
You should combine the same styles for big and small forms. 
<img width="1508" alt="image" src="https://user-images.githubusercontent.com/99403198/222127837-84ceb061-8cf1-4187-8827-21d145d64a18.png">